ilc/server/tailor/configs-injector.js

```diff
@@ -101,6 +103,19 @@ module.exports = class ConfigsInjector {
         return `<script type="spa-config">${JSON.stringify(_.omit(registryConf, ['templates']))}</script>`;
     };
 
+    /**
+     * This style is needed to avoid a flash of unstyled content (FOUC) on Firefox
+     * 
+     * @see {@link https://gist.github.com/electrotype/7960ddcc44bc4aea07a35603d1c41cb0#file-fouc-fix-md}
+     * @see {@link https://stackoverflow.com/questions/952861/targeting-only-firefox-with-css}
+     */
+    #hideHTMLtoAvoidFOUC = (hideHTML = true) => {
+        const visibility = hideHTML ? 'hidden' : 'visible';
+        const opacity = hideHTML ? 0 : 1;
+
+        return `<style>@supports (-moz-appearance:none) { html { visibility: ${visibility}; opacity: ${opacity}; }}</style>`;
```

Comment:
I reproduced it on Chrome
I guess we should remove @supports and leave it for all browsers